realm/src/main/java/io/realm/Realm.java

```diff
@@ -955,6 +955,35 @@ boolean contains(Class<?> clazz) {
         return new RealmResults<E>(this, tableView, clazz);
     }
 
+    /**
+     * Get all objects of a specific Class sorted by specific field names.
+     *
+     * @param clazz the Class to get objects of.
+     * @param fieldNames an array of field names to sort by.
+     * @param sortAscending sort ascending if SORT_ORDER_ASCENDING, sort descending if SORT_ORDER_DESCENDING.
+     * @return  A sorted RealmResults containing the objects.
+     * @throws java.lang.IllegalArgumentException if a field name does not exist.
+     */
+    public <E extends RealmObject> RealmResults<E> allObjects(Class<E> clazz, String fieldNames[], boolean sortAscending) {
```

Comment:
It looks a bit weird with the String array. I would either go with a List<String> or varargs, or both